test/testClients.js

```diff
@@ -41,6 +41,7 @@ test('Test generate Telesign headers with POST', (assert) => {
     const nonce = 'A1592C6F-E384-4CDB-BC42-C3AB970369E9';
     const resource = '/v1/resource';
     const bodyParamsURLEncoded = 'test=param';
+    const contentType = "application/x-www-form-urlencoded";
```

Comment:
Doesn't look like you have any test coverage for contentType = "application/json".